src/fastfield/readers.rs

```diff
@@ -18,16 +19,22 @@ use crate::space_usage::{FieldUsage, PerFieldSpaceUsage};
 #[derive(Clone)]
 pub struct FastFieldReaders {
     columnar: Arc<ColumnarReader>,
+    schema: Schema,
 }
 
 impl FastFieldReaders {
-    pub(crate) fn open(fast_field_file: FileSlice) -> io::Result<FastFieldReaders> {
+    pub(crate) fn open(fast_field_file: FileSlice, schema: Schema) -> io::Result<FastFieldReaders> {
         let columnar = Arc::new(ColumnarReader::open(fast_field_file)?);
-        Ok(FastFieldReaders { columnar })
+        Ok(FastFieldReaders { columnar, schema })
     }
 
-    pub(crate) fn columnar(&self) -> &ColumnarReader {
-        self.columnar.as_ref()
+    fn resolve_field(&self, column_name: &str) -> Option<String> {
+        let default_field_opt: Option<Field> = if cfg!(feature = "quickwit") {
+            self.schema.get_field("_dynamic").ok()
```

Comment:
I think it would be better if this was a configurable parameter. Not sure if that would add too much complexity though

Reply:
I agree...but let's do this in another ticket.
https://github.com/quickwit-oss/tantivy/issues/1918